.github/workflows/basic.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           -DUMF_FORMAT_CODE_STYLE=OFF
           -DUMF_DEVELOPER_MODE=ON
           -DUMF_BUILD_LIBUMF_POOL_JEMALLOC=ON
+          -DUMF_BUILD_LIBUMF_POOL_DISJOINT=ON
           ${{matrix.pool_tracking}}
 
       - name: Build UMF
```

Comment:
Could you add (as the last one) the new step for testing `make install`?
`- name: Install UMF`

Reply:
I think this should be a part of separate PR where we add some tests for the installation. There is a tasks for that in our JIRA [UMFW-176](https://jira.devtools.intel.com/browse/UMFW-176)

Reply:
OK